addons/parachute/functions/fnc_checkCutParachute.sqf

```diff
@@ -0,0 +1,18 @@
+#include "script_component.hpp"
+/*
+	Name: check Cut Parachute
+
+	Author: joko // Jonas
+
+	Description:
+	Check if Parachute can be Cuted
+
+	Parameters:
+	0: Object
+
+	Returns:
+	Boolean
+*/
+_unit = _this select 0;
+
+if (animationState _unit == "para_pilot" && !(GETVAR(_unit,chuteIsCut,false))) then {true} else {false};
```

Comment:
> Probably better to use this instead:

Yes please! Those macros are stupidly complicated. I've just read the definition of `GETGVAR` and it won't work either.